parm/atm/obs/config/amsua_n19.yaml

```diff
@@ -44,7 +44,7 @@ obs bias:
       inflation:
         ratio: 1.1
         ratio for small dataset: 2.0
-    output file: $(BIAS_DIR)/$(OBS_PREFIX).amsua_n19.satbias.$(OBS_DATE).nc4
+    output file: $(BIAS_DIR)/$(OBS_PREFIX).amsua_n19.satbias_cov.$(OBS_DATE).nc4
```

Comment:
This comment is not for the output satbias_cov file but rather for the input file.   I think the input covariance file should be `$(BIAS_DIR)/$(BIAS_PREFIX).amsua_n19.satbias_cov.$(BIAS_DATE).nc4`

Reply:
Yes, I opened an issue here https://github.com/JCSDA-internal/ufo/issues/1964 I am not sure about if they are supposed to be separate or not, as the converter suggests they are all in one file...

Reply:
OK.   Good to open [ufo #1964](https://github.com/JCSDA-internal/ufo/issues/1964) to get clarification.